pom.xml

```diff
@@ -23,6 +23,7 @@
         <netty.version>4.1.78.Final</netty.version>
         <mockito.version>3.5.13</mockito.version>
         <prometheus.version>0.16.0</prometheus.version>
+        <dropwizard-metrics.version>3.1.5</dropwizard-metrics.version> <!-- This old version is used by Cassandra 4.x -->
```

Comment:
I don't know if it matters, but if there are different versions in use by different C*/DSE versions, we can move this property to the C*/DSE specific Maven sub-module pom.

Reply:
:+1: